src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const th = document.createElement('th');
+
+th.textContent = table.tHead.rows[0].cells[1].textContent;
+table.tHead.rows[0].cells[3].after(th);
+table.tFoot.rows[0].cells[3].after(th.cloneNode(true));
```

Comment:
Similar to the previous comment, ensure that `table.tFoot` and `table.tFoot.rows[0].cells[3]` exist before attempting to insert after it.